gamersnet_backend/persistence/posts.js

```diff
@@ -54,5 +68,39 @@ async function addPost(userID, description, gameName, numPlayers, gameTimeUTC, d
   })
 }
 
+/**
+ * updates the specified post(by post id)
+ * All parameters should be string type to be consistent and avoid confusion
+ * @param {*} userID userId of the owner of this post
+ * @param {*} description 
+ * @param {*} gameName 
+ * @param {*} numPlayers 
+ * @param {*} gameTimeUTC when it will be played
+ * @param {*} duration how long will it be played
+ * @param {*} location location of game
+ */
+ async function updatePost_db(_id, description, gameName, numPlayers, gameTimeUTC, duration, location) {
```

Comment:
Would suggest a name such as `updatePostDB` so that it aligns with coding standards. Not changing it myself since this branch might still be WIP